src/game/UICmdList.h

```diff
@@ -0,0 +1,60 @@
+// UICmdListDlg.h: interface for the CUICmdListDlg class.
+//
+//////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "N3Base/N3UIBase.h"
+#include "N3Base/N3UIButton.h"
+#include "N3Base/N3UIList.h"
+#include "N3Base/N3UIEdit.h"
+
+//////////////////////////////////////////////////////////////////////
+
+class CUICmdListDlg : public CN3UIBase {
+    bool               m_bDestoyDlgAlive;
+    class CUICmdEdit * m_pUICmdEdit;
```

Comment:
In its current state this forward declaration is wrong. You named it `CUICmdEditDlg` (which is unofficial), so if that's the name you're sticking with, it should be forward declared as such as well.

If that gets changed though, of course it's fine to leave.